sylvia-derive/src/message.rs

```diff
@@ -682,6 +682,24 @@ impl<'a> MsgVariants<'a> {
                     querier: &'a #sylvia ::cw_std::QuerierWrapper<'a, C>,
                 }
 
+                impl<'a, C: #sylvia ::cw_std::CustomQuery> BoundQuerier<'a, C> {
+                    pub fn querier(&self) -> &'a #sylvia ::cw_std::QuerierWrapper<'a, C> {
+                        self.querier
+                    }
+
+                    pub fn contract(&self) -> &'a #sylvia ::cw_std::Addr {
+                        self.contract
+                    }
+
+                    pub fn new(contract: &'a #sylvia ::cw_std::Addr, querier: &'a #sylvia ::cw_std::QuerierWrapper<'a, C>) -> Self {
+                        Self {contract, querier}
+                    }
+
+                    pub fn borrowed(&self, querier: &'a #sylvia ::cw_std::QuerierWrapper<'a, C>) -> Self {
+                        Self::new(self.contract, querier)
+                    }
```

Comment:
Ok, this one is on me as I messed up signature, but we want this to be:

```rust
pub fn new(contract: &'a #sylvia ::cw_std::Addr, querier: &'a #sylvia ::cw_std::QuerierWrapper<'a, C>) -> Self {
    Self {contract, querier}
}
```

So basically, your `new` should be called borrowed.